app/Decsys/Controllers/InstancesController.cs

```diff
@@ -33,7 +30,18 @@ public class InstancesController : ControllerBase
         }
 
         [HttpGet("{instanceId}/results")]
-        public IActionResult Results(int instanceId, string? type = "summary")
+        [SwaggerOperation("Get results for a Survey Instance.")]
+        [SwaggerResponse(200, "The requested full results of an Survey Instance.", 
```

Comment:
Swagger only currently generated the second return here. Looks to be although it's possible in the OA3 spec (`oneOf`), it's not implemented in the dotnet library.